bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElectionGuard.Encryption.csproj

```diff
@@ -48,32 +48,14 @@
     <ElectionGuardLibs>..\..\..\..\build\libs</ElectionGuardLibs>
   </PropertyGroup>
   <ItemGroup Label="C++ Built Libraries">
-    <None Name="Windows MSVC x86"
-          Visible="false"
-          Include="$(ElectionGuardLibs)\msvc\Win32\src\$(Configuration)\*.dll*"
-          Pack="true"
-          PackagePath="runtimes\win-x86\native">
+    <None Name="Windows MSVC x86" Visible="false" Include="$(ElectionGuardLibs)\msvc\Win32\src\$(Configuration)\*.dll*" Pack="true" PackagePath="runtimes\win-x86\native">

```

Comment:
Not a big fan of what visual studio does to these files, but this is probably bound to happen.